src/main/java/de/tum/cit/aet/artemis/programming/service/localvc/LocalVCInfoContributor.java

```diff
@@ -28,11 +30,8 @@ public class LocalVCInfoContributor implements InfoContributor {
     @Value("${server.url}")
     private String artemisServerUrl;
 
-    @Value("${artemis.version-control.use-version-control-access-token:false}")
-    private boolean useVcsAccessToken;
-
-    @Value("${artemis.version-control.show-clone-url-without-token:true}")
```

Comment:
`application-localvc.yml` still refers to the old now unused values, rather than the single new one.

Reply:
oh good point

Reply:
I fixed the ymls :+1: